info.json

```diff
@@ -0,0 +1,52 @@
+{
+    "name": "fortiSOARForSlack",
+    "version": "1.0.0",
+    "fsrMinCompatibility": "7.3.1",
+    "type": "solutionpack",
+    "local": true,
+    "label": "FortiSOAR For Slack",
+    "dependencies": [
+        {
+            "name": "sOARFramework",
+            "type": "solutionpack",
+            "label": "SOAR Framework",
+            "minVersion": null
+        }
+    ],
+    "prerequisite": null,
+    "publisher": "Community",
+    "description": "FortiSOAR for Slack application allows teams to leverage the power of FortiSOAR as part of their daily communications.",
+    "help": null,
```

Comment:
please add doc link, also add "certified": "false", if this is uncertified release